src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -107,6 +111,14 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
      */
     DockerRegistryCredentials registryCredentials
 
+    /**
+     * Output file containing the image ID of the built image. 
+     * Defaults to "$buildDir/.docker/$taskpath-imageId.txt".
```

Comment:
The logic turn the `:` character into an underscore. This statement isn't fully correct.